Linting results in "panic: interface conversion: interface {} is nil, not string"

Hi,

I can't figure out why linting this file results in panic/crash:

`dashboard-linter lint dashboards/sre/sre-kubernetes-events.json -c grafana-dashboard.lint`. 
```
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
github.com/grafana/dashboard-linter/lint.(*Template).UnmarshalJSON(0x14000796720, {0x140007b96d8, 0x321, 0x3f2})
        /Users/robin/go/pkg/mod/github.com/grafana/dashboard-linter@v0.0.0-20230904151122-943fab9bd36b/lint/lint.go:80 +0x37c
encoding/json.(*decodeState).object(0x14000110090, {0x1055503e0?, 0x14000796720?, 0x104b38a28?})
        /opt/homebrew/Cellar/go/1.20.5/libexec/src/encoding/json/decode.go:613 +0x650
encoding/json.(*decodeState).value(0x14000110090, {0x1055503e0?, 0x14000796720?, 0x6?})
        /opt/homebrew/Cellar/go/1.20.5/libexec/src/encoding/json/decode.go:374 +0x40
encoding/json.(*decodeState).array(0x14000110090, {0x1053ca420?, 0x14000112790?, 0x1?})
        /opt/homebrew/Cellar/go/1.20.5/libexec/src/encoding/json/decode.go:562 +0x58c
encoding/json.(*decodeState).value(0x14000110090, {0x1053ca420?, 0x14000112790?, 0x4?})
        /opt/homebrew/Cellar/go/1.20.5/libexec/src/encoding/json/decode.go:364 +0x70
encoding/json.(*decodeState).object(0x14000110090, {0x105454820?, 0x14000112790?, 0x104b3a004?})
        /opt/homebrew/Cellar/go/1.20.5/libexec/src/encoding/json/decode.go:775 +0xb34
encoding/json.(*decodeState).value(0x14000110090, {0x105454820?, 0x14000112790?, 0xa?})
        /opt/homebrew/Cellar/go/1.20.5/libexec/src/encoding/json/decode.go:374 +0x40
encoding/json.(*decodeState).object(0x14000110090, {0x1054626c0?, 0x14000112780?, 0x104b464fc?})
        /opt/homebrew/Cellar/go/1.20.5/libexec/src/encoding/json/decode.go:775 +0xb34
encoding/json.(*decodeState).value(0x14000110090, {0x1054626c0?, 0x14000112780?, 0x104b453fc?})
        /opt/homebrew/Cellar/go/1.20.5/libexec/src/encoding/json/decode.go:374 +0x40
encoding/json.(*decodeState).unmarshal(0x14000110090, {0x1054626c0?, 0x14000112780?})
        /opt/homebrew/Cellar/go/1.20.5/libexec/src/encoding/json/decode.go:181 +0x184
encoding/json.Unmarshal({0x140007b4000, 0x5ac9, 0x5aca}, {0x1054626c0, 0x14000112780})
        /opt/homebrew/Cellar/go/1.20.5/libexec/src/encoding/json/decode.go:108 +0xf4
github.com/grafana/dashboard-linter/lint.NewDashboard({0x140007b4000, 0x5ac9, 0x5aca})
        /Users/robin/go/pkg/mod/github.com/grafana/dashboard-linter@v0.0.0-20230904151122-943fab9bd36b/lint/lint.go:287 +0x70
main.glob..func2(0x105c28080?, {0x1400010f4d0?, 0x1?, 0x3?})
        /Users/robin/go/pkg/mod/github.com/grafana/dashboard-linter@v0.0.0-20230904151122-943fab9bd36b/main.go:55 +0x12c
github.com/spf13/cobra.(*Command).execute(0x105c28080, {0x1400010f470, 0x3, 0x3})
        /Users/robin/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x5c8
github.com/spf13/cobra.(*Command).ExecuteC(0x105c28640)
        /Users/robin/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x35c
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/robin/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
main.main()
        /Users/robin/go/pkg/mod/github.com/grafana/dashboard-linter@v0.0.0-20230904151122-943fab9bd36b/main.go:176 +0x28
```
grafana-dashboard.lint:  
```
exclusions:
  template-job-rule:
  template-instance-rule:
  template-label-promql-rule:
  template-datasource-rule:
  panel-title-description-rule:
  panel-units-rule:
  target-promql-rule:
  target-rate-interval-rule:
  target-job-rule:
  target-instance-rule:

```
dashboards/sre/sre-kubernetes-events.json:  
```json
{
  "annotations": {
    "list": [
      {
        "builtIn": 1,
        "datasource": {
          "type": "grafana",
          "uid": "-- Grafana --"
        },
        "enable": true,
        "hide": true,
        "iconColor": "rgba(0, 211, 255, 1)",
        "name": "Annotations & Alerts",
        "type": "dashboard"
      }
    ]
  },
  "editable": true,
  "fiscalYearStartMonth": 0,
  "graphTooltip": 0,
  "links": [
    {
      "asDropdown": true,
      "icon": "external link",
      "includeVars": false,
      "keepTime": true,
      "tags": [
        "sre"
      ],
      "targetBlank": false,
      "title": "SRE dashboards",
      "tooltip": "",
      "type": "dashboards",
      "url": ""
    }
  ],
  "liveNow": false,
  "panels": [
    {
      "gridPos": {
        "h": 1,
        "w": 24,
        "x": 0,
        "y": 0
      },
      "id": 13,
      "title": "Kubernetes Cluster",
      "type": "row"
    },
    {
      "datasource": {
        "type": "prometheus",
        "uid": "${DS_GMP}"
      },
      "fieldConfig": {
        "defaults": {
          "color": {
            "mode": "thresholds"
          },
          "custom": {
            "axisCenteredZero": false,
            "axisColorMode": "text",
            "axisLabel": "",
            "axisPlacement": "auto",
            "barAlignment": 0,
            "drawStyle": "bars",
            "fillOpacity": 100,
            "gradientMode": "hue",
            "hideFrom": {
              "legend": false,
              "tooltip": false,
              "viz": false
            },
            "lineInterpolation": "linear",
            "lineWidth": 1,
            "pointSize": 1,
            "scaleDistribution": {
              "type": "linear"
            },
            "showPoints": "auto",
            "spanNulls": false,
            "stacking": {
              "group": "A",
              "mode": "none"
            },
            "thresholdsStyle": {
              "mode": "off"
            }
          },
          "mappings": [],
          "thresholds": {
            "mode": "absolute",
            "steps": [
              {
                "color": "yellow",
                "value": null
              }
            ]
          }
        },
        "overrides": []
      },
      "gridPos": {
        "h": 8,
        "w": 12,
        "x": 0,
        "y": 1
      },
      "id": 15,
      "interval": "$interval",
      "options": {
        "legend": {
          "calcs": [],
          "displayMode": "list",
          "placement": "bottom",
          "showLegend": true
        },
        "tooltip": {
          "mode": "single",
          "sort": "none"
        }
      },
      "targets": [
        {
          "datasource": {
            "type": "prometheus",
            "uid": "${DS_GMP}"
          },
          "editorMode": "code",
          "expr": "sum by(severity)(increase(logging_googleapis_com:byte_count{monitored_resource=\"k8s_cluster\",severity!=\"INFO\",log=\"events\"}[1m]))",
          "legendFormat": "__auto",
          "range": true,
          "refId": "A"
        }
      ],
      "title": "Kubernetes Cluster Events Count",
      "type": "timeseries"
    },
    {
      "datasource": {
        "type": "googlecloud-logging-datasource",
        "uid": "${DS_GCP_LOGGING}"
      },
      "fieldConfig": {
        "defaults": {
          "color": {
            "mode": "thresholds"
          },
          "custom": {
            "align": "auto",
            "cellOptions": {
              "type": "auto"
            },
            "inspect": false
          },
          "mappings": [],
          "thresholds": {
            "mode": "absolute",
            "steps": [
              {
                "color": "green",
                "value": null
              },
              {
                "color": "red",
                "value": 80
              },
              {
                "color": "#EAB839",
                "value": 90
              }
            ]
          }
        },
        "overrides": [
          {
            "matcher": {
              "id": "byName",
              "options": "Time"
            },
            "properties": [
              {
                "id": "custom.width",
                "value": 190
              }
            ]
          },
          {
            "matcher": {
              "id": "byName",
              "options": "jsonPayload.involvedObject.apiVersion"
            },
            "properties": [
              {
                "id": "custom.width",
                "value": 371
              }
            ]
          }
        ]
      },
      "gridPos": {
        "h": 8,
        "w": 12,
        "x": 12,
        "y": 1
      },
      "id": 14,
      "options": {
        "cellHeight": "sm",
        "footer": {
          "countRows": false,
          "fields": "",
          "reducer": [
            "sum"
          ],
          "show": false
        },
        "frameIndex": 2,
        "showHeader": true,
        "sortBy": []
      },
      "pluginVersion": "9.5.1",
      "targets": [
        {
          "datasource": {
            "type": "googlecloud-logging-datasource",
            "uid": "${DS_GCP_LOGGING}"
          },
          "projectId": "${GCP_PROJECT_ID}",
          "queryText": "resource.type=\"k8s_cluster\"\nlog_name=~\"projects/gcp-dev-.*/logs/events\"",
          "refId": "A"
        }
      ],
      "title": "Kubernetes Cluster Event Logs",
      "transformations": [
        {
          "id": "seriesToRows",
          "options": {}
        },
        {
          "id": "organize",
          "options": {
            "excludeByName": {
              "Metric": true
            },
            "indexByName": {},
            "renameByName": {}
          }
        }
      ],
      "type": "table"
    },
    {
      "collapsed": false,
      "gridPos": {
        "h": 1,
        "w": 24,
        "x": 0,
        "y": 9
      },
      "id": 10,
      "panels": [],
      "title": "Kubernetes Container",
      "type": "row"
    },
    {
      "datasource": {
        "type": "prometheus",
        "uid": "${DS_GMP}"
      },
      "fieldConfig": {
        "defaults": {
          "color": {
            "mode": "palette-classic"
          },
          "custom": {
            "axisCenteredZero": false,
            "axisColorMode": "text",
            "axisLabel": "",
            "axisPlacement": "auto",
            "barAlignment": 0,
            "drawStyle": "bars",
            "fillOpacity": 100,
            "gradientMode": "hue",
            "hideFrom": {
              "legend": false,
              "tooltip": false,
              "viz": false
            },
            "lineInterpolation": "linear",
            "lineWidth": 0,
            "pointSize": 5,
            "scaleDistribution": {
              "type": "linear"
            },
            "showPoints": "auto",
            "spanNulls": false,
            "stacking": {
              "group": "A",
              "mode": "none"
            },
            "thresholdsStyle": {
              "mode": "off"
            }
          },
          "mappings": [],
          "thresholds": {
            "mode": "absolute",
            "steps": [
              {
                "color": "green",
                "value": null
              },
              {
                "color": "red",
                "value": 80
              }
            ]
          }
        },
        "overrides": [
          {
            "matcher": {
              "id": "byValue",
              "options": {
                "op": "gte",
                "reducer": "allIsZero",
                "value": 0
              }
            },
            "properties": [
              {
                "id": "custom.hideFrom",
                "value": {
                  "legend": true,
                  "tooltip": true,
                  "viz": true
                }
              }
            ]
          }
        ]
      },
      "gridPos": {
        "h": 7,
        "w": 24,
        "x": 0,
        "y": 10
      },
      "id": 9,
      "interval": "$interval",
      "options": {
        "legend": {
          "calcs": [
            "mean",
            "max"
          ],
          "displayMode": "table",
          "placement": "right",
          "showLegend": true,
          "sortBy": "Mean",
          "sortDesc": true
        },
        "tooltip": {
          "mode": "multi",
          "sort": "desc"
        }
      },
      "pluginVersion": "9.5.1",
      "targets": [
        {
          "datasource": {
            "type": "prometheus",
            "uid": "${DS_GMP}"
          },
          "editorMode": "code",
          "exemplar": false,
          "expr": "sum by(namespace_name,container_name,severity)(increase(logging_googleapis_com:byte_count{monitored_resource=\"k8s_container\",severity!=\"INFO\",namespace_name=~\"${namespace}\"}[1m]))",
          "format": "time_series",
          "instant": false,
          "interval": "",
          "key": "Q-74fa07a5-79da-4e61-8ed4-5e23a5e9a935-0",
          "legendFormat": "{{namespace_name}}/{{container_name}}/{{severity}}",
          "range": true,
          "refId": "D"
        }
      ],
      "title": "Kubernetes Container Events Count",
      "type": "timeseries"
    },
    {
      "datasource": {
        "type": "googlecloud-logging-datasource",
        "uid": "${DS_GCP_LOGGING}"
      },
      "gridPos": {
        "h": 13,
        "w": 12,
        "x": 0,
        "y": 17
      },
      "id": 11,
      "options": {
        "dedupStrategy": "none",
        "enableLogDetails": true,
        "prettifyLogMessage": false,
        "showCommonLabels": false,
        "showLabels": false,
        "showTime": true,
        "sortOrder": "Descending",
        "wrapLogMessage": false
      },
      "targets": [
        {
          "bucketId": "",
          "datasource": {
            "type": "googlecloud-logging-datasource",
            "uid": "${DS_GCP_LOGGING}"
          },
          "projectId": "${GCP_PROJECT_ID}",
          "queryText": "severity=ERROR\nresource.type=\"k8s_container\"",
          "refId": "A",
          "viewId": ""
        }
      ],
      "title": "Kubernetes Container Error Logs",
      "type": "logs"
    },
    {
      "datasource": {
        "type": "googlecloud-logging-datasource",
        "uid": "${DS_GCP_LOGGING}"
      },
      "gridPos": {
        "h": 13,
        "w": 12,
        "x": 12,
        "y": 17
      },
      "id": 12,
      "options": {
        "dedupStrategy": "none",
        "enableLogDetails": true,
        "prettifyLogMessage": false,
        "showCommonLabels": false,
        "showLabels": false,
        "showTime": true,
        "sortOrder": "Descending",
        "wrapLogMessage": false
      },
      "targets": [
        {
          "bucketId": "",
          "datasource": {
            "type": "googlecloud-logging-datasource",
            "uid": "${DS_GCP_LOGGING}"
          },
          "projectId": "${GCP_PROJECT_ID}",
          "queryText": "severity=WARNING\nresource.type=\"k8s_container\"",
          "refId": "A",
          "viewId": ""
        }
      ],
      "title": "Kubernetes Container Warning Logs",
      "type": "logs"
    },
    {
      "collapsed": false,
      "gridPos": {
        "h": 1,
        "w": 24,
        "x": 0,
        "y": 30
      },
      "id": 8,
      "panels": [],
      "title": "Kubernetes Cluster Autoscaling",
      "type": "row"
    },
    {
      "datasource": {
        "type": "prometheus",
        "uid": "${DS_PROMETHEUS}"
      },
      "fieldConfig": {
        "defaults": {
          "color": {
            "mode": "palette-classic"
          },
          "custom": {
            "axisCenteredZero": false,
            "axisColorMode": "text",
            "axisLabel": "",
            "axisPlacement": "auto",
            "barAlignment": 0,
            "drawStyle": "line",
            "fillOpacity": 100,
            "gradientMode": "hue",
            "hideFrom": {
              "legend": false,
              "tooltip": false,
              "viz": false
            },
            "lineInterpolation": "linear",
            "lineWidth": 1,
            "pointSize": 5,
            "scaleDistribution": {
              "type": "linear"
            },
            "showPoints": "auto",
            "spanNulls": false,
            "stacking": {
              "group": "A",
              "mode": "none"
            },
            "thresholdsStyle": {
              "mode": "off"
            }
          },
          "decimals": 0,
          "mappings": [],
          "thresholds": {
            "mode": "absolute",
            "steps": [
              {
                "color": "green"
              },
              {
                "color": "red",
                "value": 80
              }
            ]
          }
        },
        "overrides": []
      },
      "gridPos": {
        "h": 7,
        "w": 12,
        "x": 0,
        "y": 31
      },
      "id": 6,
      "options": {
        "legend": {
          "calcs": [],
          "displayMode": "list",
          "placement": "bottom",
          "showLegend": true
        },
        "tooltip": {
          "mode": "single",
          "sort": "none"
        }
      },
      "targets": [
        {
          "datasource": {
            "type": "prometheus",
            "uid": "${DS_PROMETHEUS}"
          },
          "editorMode": "code",
          "expr": "sum(kube_node_info)",
          "legendFormat": "Nodes",
          "range": true,
          "refId": "A"
        }
      ],
      "title": "Kubernetes Cluster Node Scaling",
      "type": "timeseries"
    },
    {
      "datasource": {
        "type": "googlecloud-logging-datasource",
        "uid": "${DS_GCP_LOGGING}"
      },
      "gridPos": {
        "h": 7,
        "w": 12,
        "x": 12,
        "y": 31
      },
      "id": 7,
      "options": {
        "dedupStrategy": "none",
        "enableLogDetails": true,
        "prettifyLogMessage": false,
        "showCommonLabels": false,
        "showLabels": false,
        "showTime": true,
        "sortOrder": "Descending",
        "wrapLogMessage": false
      },
      "targets": [
        {
          "datasource": {
            "type": "googlecloud-logging-datasource",
            "uid": "${DS_GCP_LOGGING}"
          },
          "projectId": "${GCP_PROJECT_ID}",
          "queryText": "resource.type=\"k8s_cluster\"\nlogName=~\"projects/gcp-dev-.*/logs/container.googleapis.com%2Fcluster-autoscaler-visibility\" severity>=DEFAULT\n( \"decision\" NOT \"noDecisionStatus\" )",
          "refId": "A"
        }
      ],
      "title": "Kubernetes Cluster Autoscaler Logs",
      "transformations": [
        {
          "id": "extractFields",
          "options": {
            "format": "kvp",
            "keepTime": false,
            "replace": false,
            "source": "content"
          }
        }
      ],
      "type": "logs"
    },
    {
      "collapsed": false,
      "gridPos": {
        "h": 1,
        "w": 24,
        "x": 0,
        "y": 38
      },
      "id": 4,
      "panels": [],
      "title": "Kubernetes Cluster Upgrades",
      "type": "row"
    },
    {
      "datasource": {
        "type": "googlecloud-logging-datasource",
        "uid": "${DS_GCP_LOGGING}"
      },
      "gridPos": {
        "h": 7,
        "w": 12,
        "x": 0,
        "y": 39
      },
      "id": 2,
      "maxDataPoints": 9999999999,
      "options": {
        "dedupStrategy": "none",
        "enableLogDetails": true,
        "prettifyLogMessage": false,
        "showCommonLabels": false,
        "showLabels": false,
        "showTime": true,
        "sortOrder": "Descending",
        "wrapLogMessage": false
      },
      "targets": [
        {
          "bucketId": "",
          "datasource": {
            "type": "googlecloud-logging-datasource",
            "uid": "${DS_GCP_LOGGING}"
          },
          "projectId": "${GCP_PROJECT_ID}",
          "queryText": "protoPayload.methodName=\"google.container.internal.ClusterManagerInternal.UpdateClusterInternal\"\nresource.type=\"gke_cluster\"",
          "refId": "A",
          "viewId": ""
        }
      ],
      "title": "GKE upgrades - Master/Controlplane",
      "type": "logs"
    },
    {
      "datasource": {
        "type": "googlecloud-logging-datasource",
        "uid": "${DS_GCP_LOGGING}"
      },
      "gridPos": {
        "h": 7,
        "w": 12,
        "x": 12,
        "y": 39
      },
      "id": 3,
      "options": {
        "dedupStrategy": "none",
        "enableLogDetails": true,
        "prettifyLogMessage": false,
        "showCommonLabels": false,
        "showLabels": false,
        "showTime": true,
        "sortOrder": "Descending",
        "wrapLogMessage": false
      },
      "targets": [
        {
          "bucketId": "",
          "datasource": {
            "type": "googlecloud-logging-datasource",
            "uid": "${DS_GCP_LOGGING}"
          },
          "projectId": "${GCP_PROJECT_ID}",
          "queryText": "protoPayload.methodName=\"google.container.internal.ClusterManagerInternal.UpdateClusterInternal\"\nresource.type=\"gke_nodepool\"",
          "refId": "A",
          "viewId": ""
        }
      ],
      "title": "GKE upgrades - NodePool",
      "type": "logs"
    },
    {
      "datasource": {
        "type": "prometheus",
        "uid": "${DS_PROMETHEUS}"
      },
      "fieldConfig": {
        "defaults": {
          "color": {
            "mode": "thresholds"
          },
          "custom": {
            "fillOpacity": 70,
            "lineWidth": 0,
            "spanNulls": false
          },
          "mappings": [],
          "thresholds": {
            "mode": "absolute",
            "steps": [
              {
                "color": "green"
              },
              {
                "color": "red",
                "value": 80
              }
            ]
          }
        },
        "overrides": []
      },
      "gridPos": {
        "h": 4,
        "w": 12,
        "x": 0,
        "y": 46
      },
      "id": 5,
      "options": {
        "alignValue": "left",
        "legend": {
          "displayMode": "list",
          "placement": "bottom",
          "showLegend": true
        },
        "mergeValues": true,
        "rowHeight": 0.9,
        "showValue": "auto",
        "tooltip": {
          "mode": "single",
          "sort": "none"
        }
      },
      "targets": [
        {
          "datasource": {
            "type": "prometheus",
            "uid": "${DS_PROMETHEUS}"
          },
          "editorMode": "code",
          "expr": "sum(kube_node_info) by (kubelet_version)",
          "legendFormat": "__auto",
          "range": true,
          "refId": "A"
        }
      ],
      "title": "Kubernetes versions",
      "type": "state-timeline"
    }
  ],
  "refresh": "",
  "schemaVersion": 38,
  "style": "dark",
  "tags": [
    "sre",
    "kubernetes"
  ],
  "templating": {
    "list": [
      {
        "allValue": "",
        "current": {
          "selected": true,
          "text": [
            "All"
          ],
          "value": [
            "$__all"
          ]
        },
        "datasource": {
          "type": "prometheus",
          "uid": "${DS_PROMETHEUS}"
        },
        "definition": "label_values(namespace)",
        "hide": 0,
        "includeAll": true,
        "label": "Namespace",
        "multi": true,
        "name": "namespace",
        "options": [],
        "query": {
          "query": "label_values(namespace)",
          "refId": "PrometheusVariableQueryEditor-VariableQuery"
        },
        "refresh": 2,
        "regex": "",
        "skipUrlSync": false,
        "sort": 5,
        "type": "query"
      },
      {
        "current": {
          "selected": false,
          "text": "1h",
          "value": "1h"
        },
        "hide": 0,
        "includeAll": false,
        "label": "Interval",
        "multi": false,
        "name": "interval",
        "options": [
          {
            "selected": false,
            "text": "30s",
            "value": "30s"
          },
          {
            "selected": false,
            "text": "60s",
            "value": "60s"
          },
          {
            "selected": false,
            "text": "10m",
            "value": "10m"
          },
          {
            "selected": true,
            "text": "1h",
            "value": "1h"
          },
          {
            "selected": false,
            "text": "12h",
            "value": "12h"
          },
          {
            "selected": false,
            "text": "1d",
            "value": "1d"
          }
        ],
        "query": "30s,60s,10m,1h,12h,1d",
        "queryValue": "",
        "skipUrlSync": false,
        "type": "custom"
      },
      {
        "current": {
          "selected": false,
          "text": "victoria-metrics",
          "value": "victoria-metrics"
        },
        "hide": 2,
        "includeAll": false,
        "multi": false,
        "name": "DS_PROMETHEUS",
        "options": [],
        "query": "prometheus",
        "refresh": 1,
        "regex": "/victoria-metrics/",
        "skipUrlSync": false,
        "type": "datasource"
      },
      {
        "current": {
          "selected": false,
          "text": "google-managed-prometheus",
          "value": "google-managed-prometheus"
        },
        "hide": 2,
        "includeAll": false,
        "label": "",
        "multi": false,
        "name": "DS_GMP",
        "options": [],
        "query": "prometheus",
        "refresh": 1,
        "regex": "/google/",
        "skipUrlSync": false,
        "type": "datasource"
      },
      {
        "current": {
          "selected": false,
          "text": "Google Cloud Logging",
          "value": "Google Cloud Logging"
        },
        "hide": 2,
        "includeAll": false,
        "multi": false,
        "name": "DS_GCP_LOGGING",
        "options": [],
        "query": "googlecloud-logging-datasource",
        "refresh": 1,
        "regex": "/Google Cloud Logging/",
        "skipUrlSync": false,
        "type": "datasource"
      },
      {
        "current": {
          "selected": false,
          "text": "gcp-dev",
          "value": "gcp-dev"
        },
        "datasource": {
          "type": "googlecloud-logging-datasource",
          "uid": "${DS_GCP_LOGGING}"
        },
        "definition": "",
        "hide": 2,
        "includeAll": false,
        "multi": false,
        "name": "GCP_PROJECT_ID",
        "options": [],
        "query": {
          "bucketId": "",
          "loading": false,
          "projectId": "${GCP_PROJECT_ID}",
          "refId": "CloudLoggingVariableQueryEditor-VariableQuery",
          "selectedQueryType": "projects",
          "viewId": ""
        },
        "refresh": 1,
        "regex": "",
        "skipUrlSync": false,
        "sort": 0,
        "type": "query"
      }
    ]
  },
  "time": {
    "from": "now-24h",
    "to": "now"
  },
  "timepicker": {},
  "timezone": "",
  "title": "Kubernetes Events",
  "uid": "sre-kubernetes-events",
  "version": 1,
  "weekStart": ""
}

```

--- a/lint/lint.go
+++ b/lint/lint.go
@@ -77,7 +77,10 @@ func (t *Template) UnmarshalJSON(buf []byte) error {
 		case string:
 			t.Query = v
 		case map[string]interface{}:
-			t.Query = v[targetTypeQuery].(string)
+			query, ok := v[targetTypeQuery]
+			if ok {
+				t.Query = query.(string)
+			}
 		default:
 			return fmt.Errorf("invalid type for field 'query': %v", v)
 		}

--- a/lint/model_test.go
+++ b/lint/model_test.go
@@ -105,3 +105,22 @@ func TestParseTemplateValue(t *testing.T) {
 		require.Equal(t, tc.expected, actual)
 	}
 }
+
+func TestParseTemplate(t *testing.T) {
+	for _, tc := range []struct {
+		input    []byte
+		expected Template
+		err      error
+	}{
+		{
+			// NB no "query.query" field, some data source don't use this.
+			input:    []byte(`{ "type": "query", "query": {} }`),
+			expected: Template{Type: "query", RawQuery: map[string]interface{}{}},
+		},
+	} {
+		var actual Template
+		err := json.Unmarshal(tc.input, &actual)
+		require.NoError(t, err)
+		require.Equal(t, tc.expected, actual)
+	}
+}

